src/main/java/com/itschool/eventmanagment/models/dtos/EventDTO.java

```diff
@@ -41,4 +46,33 @@ public static Event mapEventDtoToEvent(EventDTO eventDTO) {
         event.setMaxParticipants(eventDTO.getMaxParticipants());
         return event;
     }
+
+    public EventDTO(Long id, String name, String location, LocalDateTime eventDateTime, Long userId, int maxParticipants, List<ParticipantDTO> registeredParticipants) {
+        this.id = id;
+        this.name = name;
+        this.location = location;
+        this.eventDateTime = eventDateTime;
+        this.userId = userId;
+        this.maxParticipants = maxParticipants;
+        this.registeredParticipants = registeredParticipants;
+    }
+
+    public EventDTO(Long id, String name, String location, LocalDateTime eventDateTime, Long userId, int maxParticipants) {
+        this.id = id;
+        this.name = name;
+        this.location = location;
+        this.eventDateTime = eventDateTime;
+        this.userId = userId;
+        this.maxParticipants = maxParticipants;
+    }
+
+    public EventDTO(String name, String location, LocalDateTime eventDateTime, Long userId, int maxParticipants) {
+        this.name = name;
+        this.location = location;
+        this.eventDateTime = eventDateTime;
+        this.userId = userId;
+        this.maxParticipants = maxParticipants;
+    }
+    public EventDTO() {
```

Comment:
Can we use a no-args constructor annotation from Lombok for this?
https://projectlombok.org/features/constructor